.gitignore

```diff
@@ -114,6 +114,7 @@ logs
 results
 node_modules
 .node_history
+package-lock.json
```

Comment:
Shouldn't be added to the .gitignore. Please use yarn as default package manager.